tests/services/components/test_external_pids_component.py

```diff
@@ -182,29 +188,6 @@ def test_unmanaged_no_required_no_partial_value(
         component.publish(identity_simple, draft=draft, record=record)
 
 
-def test_unmanaged_no_required_no_value(
```

Comment:
Removed because is not a valid use case. If it's not required, there is no "default" provider... only required ones are checked to *must have one required provider at least* (i.e. the default)